examples/passport-login/src/authentication-strategies/types.ts

```diff
@@ -0,0 +1,78 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import axios from 'axios';
+import {Profile} from 'passport';
+import {UserIdentityService} from '@loopback/authentication';
+import {User} from '../models';
+
+export type profileFunction = (
+  accessToken: string,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  done: (err?: Error | null, profile?: any) => void,
+) => void;
+
+export type VerifyFunction = (
+  accessToken: string,
+  refreshToken: string,
+  profile: Profile,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  done: (error: any, user?: any, info?: any) => void,
+) => void;
+
+export namespace PassportAuthenticationBindings {
```

Comment:
please move the bindings into  `keys.ts` file.

Reply:
agreed